samples/net/sockets/big_http_download/prj.conf

```diff
@@ -3,6 +3,7 @@ CONFIG_NEWLIB_LIBC=y
 CONFIG_MBEDTLS=y
 CONFIG_MBEDTLS_ENABLE_HEAP=y
 CONFIG_MAIN_STACK_SIZE=2176
+CONFIG_SAMPLE_BIG_HTTP_DL_NUM_ITER=3
```

Comment:
Please don't change existing settings without good motivation. Such changes and motivation should go into a separate PR (on its own). Current usecase for big_http_download is that a user can start it and use for assessing IP stack overall stability, e.g. wait and watch until it downloads 1GB total data (then one kinda assume that "IP stack in Zephyr works").